spec/controllers/ems_common_controller_spec.rb

```diff
@@ -191,6 +191,82 @@ def test_setting_few_fields
           expect(@ems.options[:image_inspector_options][:http_proxy]).to eq("example.com")
         end
       end
+
+      context "update container provider with kubevirt provider" do
```

Comment:
@masayag Would it be possible to move this test out of `ems_common`, and move it to `ems_container_controller_spec` instead? 
In fact, all the other `ems_container` specific tests before this PR also need to be moved if possible.
(but not in this PR)

Reply:
Can you also include a test for editing/updating the kubevirt provider in `ems_infra_controller_spec`?

Reply:
@AparnaKarve fixed the specs and added missing ones. 

@moolitayer @zeari please see comment about pulling ems_container related specs to ems_container_controller_spec